README.md

```diff
@@ -217,7 +217,42 @@ const args = {
 }
 ```
 
-#### Errors
+#### `stopAtPositional`
+
+When `stopAtPositional` is set to `true`, `arg` will halt parsing at the first
+positional argument.
+
+This is useful for when sub-command handling should occur, or when paired with
+`literalDashes`.
```

Comment:
I thought `literalDashes` had ceased to exist

Reply:
Good catch, did this too quickly.